src/util/src/ttimer.c

```diff
@@ -552,7 +552,7 @@ void* taosTmrInit(int maxNumOfTmrs, int resolution, int longest, const char* lab
     return NULL;
   }
 
-  strncpy(ctrl->label, label, sizeof(ctrl->label));
+  STRNCPY(ctrl->label, label, sizeof(ctrl->label));
   ctrl->label[sizeof(ctrl->label) - 1] = 0;
```

Comment:
this line could be removed after the change.

Reply:
ok